connector_dns_dnspod/readme.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Connector Dns Dnspod
+=====================
+
+This module aims to allows you to manage your DNSPod domain through Odoo.
```

Comment:
This module allows you to manage your DNSPod domain through Odoo.
It uses the service from www.dnspod.cn and synchronizes (create/delete/update)
all relevant information such as domain names and records.
This module is based on the odoo-connector tehcnology.